src/libraries/Common/src/System/Net/TlsStream.cs

```diff
@@ -14,6 +14,7 @@ internal sealed class TlsStream : NetworkStream
         private readonly string _host;
         private readonly X509CertificateCollection? _clientCertificates;
 
+#pragma warning disable SYSLIB0014 // ServicePointManager is obsolete
```

Comment:
Presumably we should use a new diagnostic ID?

Reply:
We are already using SYSLIB0014 is for the already obsolete `ServicePointManager.FindServicePoint()` methods, therefore I don't see the point of introducing a new one.  The current diagnostic message for SYSLIB0014  is `"WebRequest, HttpWebRequest, ServicePoint, WebClient are obsolete"`, which is wrong since `ServicePoint` is not yet obsolete. We probably intended to mention `ServicePointManager` instead. We should consider updating it.

Reply:
> which is wrong since ServicePoint is not yet obsolete

Why are we obsoleting ServicePointManager but not ServicePoint?

Reply:
We use `ServicePoint` in many legacy (but not yet obsolete) API-s like HttpListener, System.Net.Mail, FtpWebRequest. Where to draw the line? It's going to be a long discussion and very likely abandoned I'm afraid.

My position is to do the minimum necessary work to deliver a relatively consistent change in 7.0 that would warn users if they are touching API-s which do not do anything at all. From that perspective, I still think that even #71833 would make sense.